assets/src/edit-story/components/panels/panel/shared/title.js

```diff
@@ -193,6 +185,11 @@ function Title({
       isPrimary={isPrimary}
       isSecondary={isSecondary}
       hasResizeHandle={isResizable && !isCollapsed}
+      aria-label={title}
+      aria-expanded={!isCollapsed}
+      aria-controls={panelContentId}
+      role="button"
+      onClick={onToggle}
```

Comment:
Since we're turning `<Header>` into a button, is `<HeaderButton>` now obsolete? Can its CSS be merged into `<Header>`?

Reply:
Right, we did turn `<header>` into a button, but we might still need to keep the style `<HeaderButton>`, which was a `div` and its styling is for its children `<Heading>` and <HeaderActions>`.

Reply:
And to clarify, `<HeaderButton>` is obsolete and is no longer a `button`, instead, it is merely a `div`, we are keeping this element for styling reason.

Reply:
Yea, see its definition above: https://github.com/google/web-stories-wp/pull/5342/files#diff-a54a124ac5a73d9a8cecd8ccfb719d1cf8bff8738fa127b72f8779ad7c36bf3aR63-R70

My question is whether we can remove `<HeaderButton>` by merging its CSS into `<Header>`. 

Minimally, we can rename `HeaderButton` to `HeaderLayout` and remove `{role: 'button'}` attribute.

Reply:
Got it! And agree to remove the unnecessary layer!